package.json

```diff
@@ -20,8 +20,9 @@
     "postversion": "git add package.json package-lock.json && git commit -m $npm_package_version",
     "pretest": "npm run test:lint",
     "docker:debug": "docker compose -f compose.yaml -f compose.debug.yaml -p \"fcp-sfd-frontend\" up",
-    "docker:test": "docker compose down -v && docker compose -f compose.yaml -f compose.test.yaml run --build --rm 'fcp-sfd-frontend'",
-    "docker:test:watch": "docker compose down -v && docker compose -f compose.yaml -f compose.test.yaml -f compose.test.watch.yaml run --build --rm 'fcp-sfd-frontend'",
+    "docker:dev": "docker compose up",
+    "docker:test": "npm run lint && mkdir -p coverage 2>/dev/null || true docker compose -p fcp-sfd-frontend-test down -v && docker compose -f compose.yaml -f compose.test.yaml -p fcp-sfd-frontend-test run --build --rm fcp-sfd-frontend",
+    "docker:test:watch": "docker compose -p fcp-sfd-frontend-test down -v && docker compose -f compose.yaml -f compose.test.yaml -f compose.test.watch.yaml -p fcp-sfd-frontend-test run --build --rm fcp-sfd-frontend",
```

Comment:
There was an issue in the original setup here which prevented me from testing effectively.

By adding a "project" for the test setup, it means they will run as a new set of independent and isolated containers.

This means you can now run the application and run the tests at the same time without conflict.  Handy for TDD and seeing your changes.

Also added lint check as that wasn't included in the CI build anywhere meaning it's possible to commit unlinted code.